deploy/kubernetes/helm/che/custom-charts/che-devfile-registry/templates/ingress.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: che-devfile-registry
+  annotations:
+    kubernetes.io/ingress.class: {{ .Values.cheDevfileRegistry.ingress.class }}
```

Comment:
In the operator, the same settings are used for all ingresses: the Che one, the Keycloak one, and now the registry ones.
Does it have sense to have a distinct ingress class for each registry ?

Reply:
changed to `nginx`, same as others

Reply:
Don't you have a global `ingressClass` variable, just the same as the global `ingressSecretName` you used in my other comment ?

Reply:
The global `ingressClass` is configurable in the Operator.

Reply:
I see hardcoded `nginx` everywhere. I can create it.

Reply:
If it was hard-coded everywhere until now, you can also keep this change for another PR (at least for me).

Reply:
let me mark this conversation as resolved and let you choose the solution that is the best for you.

Reply:
yes, I'll create task issue for that

Reply:
#13927